2.7.0/done/result_quantization.md

```diff
@@ -27,42 +27,93 @@ The categories below are as follows:
+- Remove capture_pre_autograd_graph references in quantization ([#139505](https://github.com/pytorch/pytorch/pull/139505))
+
+```python
+# pytorch 2.6
+from torch._export import capture_pre_autograd_graph
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+from torch.ao.quantization.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = capture_pre_autograd_graph(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+
+# pytorch 2.7
+from torch.export import export_for_training
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+# please get xnnpack quantizer from executorch (https://github.com/pytorch/executorch/)
+from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = export_for_training(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+```
+
 ### deprecation
+- Add deprecation warning to XNNPACKQuantizer ([#144940](https://github.com/pytorch/pytorch/pull/144940))
+
+```python
+# pytorch 2.6
+from torch._export import capture_pre_autograd_graph
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+from torch.ao.quantization.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = capture_pre_autograd_graph(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+
+# pytorch 2.7
+# we also updated the export call
+from torch.export import export_for_training
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+# please get xnnpack quantizer from executorch (https://github.com/pytorch/executorch/)
+from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = export_for_training(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+```
+
+
 ### new features
+- debug handler maintain through decomposition ([#141612](https://github.com/pytorch/pytorch/pull/141612))
+- Add support for list, tuple and dict in numeric debugger ([#143882](https://github.com/pytorch/pytorch/pull/143882))
+
 ### improvements
+- Add an option keep_original_weights in _lower_to_native_backend ([#141049](https://github.com/pytorch/pytorch/pull/141049))
+- Handle meta tensors in FX quantization ([#144726](https://github.com/pytorch/pytorch/pull/144726))
+- add fp8 support to index_cuda ([#144747](https://github.com/pytorch/pytorch/pull/144747))
+- patch for block-wise quantization + pt2e ([#146946](https://github.com/pytorch/pytorch/pull/146946))
```

Comment:
be specific about the patch -- what was fixed?

Reply:
also removed, this is experimental, not intended for wider use